packages/customer-dashboard/StreetLevelOneDashboard/src/components/HeaderBar.tsx

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { Box, Image, Header, Button, Menu } from "grommet";
+import { UserSettings } from "grommet-icons";
+
+// TO MAKE BUTTONS ACTIVE WHEN ON PAGE - I image it's a simple useState thing I need to figure out
+
+export const HeaderBar = () => (
+  <Header
+    background="#000000"
+    margin="none"
+    pad={{ horizontal: "medium", vertical: "medium" }}
+    elevation="small"
+    justify="between"
+    height="15vh"
```

Comment:
That's quite a large header size, let's make it static around 50px for now and then flexbox the rest of the body